locale/zh-cn/docs/guides/domain-postmortem.md

```diff
@@ -127,16 +139,15 @@ let uid = 0;
 
 // Setting up temporary resources
 const buf = Buffer.alloc(FILESIZE);
-for (let i = 0; i < buf.length; i++)
-  buf[i] = ((Math.random() * 1e3) % 78) + 48; // Basic ASCII
+for (let i = 0; i < buf.length; i++) buf[i] = ((Math.random() * 1e3) % 78) + 48; // Basic ASCII
```

Comment:
Just wondering, don't we enforce brackets in these cases?

Reply:
I think it's the `curly` rule coming from Standard